js/plugin/popup-dialog.ts

```diff
@@ -568,7 +568,4 @@ export class BrowserModPopup extends LitElement {
       }
     `;
   }
-}
-
-if (!customElements.get("browser-mod-popup"))
-  customElements.define("browser-mod-popup", BrowserModPopup);
\ No newline at end of file
+}
```

Comment:
@thomasloven found a quirky one which I think you have seen in card-mod before with the scoped custom elements registry. It has to do with defining the standard `browser-mod-popup` customElement. Removing from being static to being dynamic is the easy fix, as it will get defined by the dynamic show popup code.